### PR TITLE
HCS fix deploy script and service files. Update version to 0.1.0-rc1

### PR DESCRIPTION
**Pull request by mike-burrage-hedera**
**Detailed description**:
- Update versions from 0.1.0-SNAPSHOT to 0.1.0-rc1
- Make service name match artifact (-hcs suffix)
- Adjust deploy script for new service name.
- Fix bug in deploy script introduced in #400. jar name is "-0.1.0", directory is "-v0.1.0". Links at the end of deploy script fail.
- Change to CircleCI - version tags for HCS will be `hcs/v0.1.0` style.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: hedera-mirror-grpc/scripts/deploy.sh [hunk 1]
```diff
@@ -1,14 +1,19 @@
 #!/bin/sh -ex
 
-name=hedera-mirror-grpc
+name=hedera-mirror-grpc-hcs
```
Comment (steven-sheehy) [inline hedera-mirror-grpc/scripts/deploy.sh]: I'm guessing previewnet has `/var/lib/hedera-mirror-importer` and this doesn't handle migration to HCS name. This would weigh in favor of not renaming services and directories.

Comment (mike-burrage-hedera) [reply hedera-mirror-grpc/scripts/deploy.sh]: Sounds good. I can change it back for grpc service. That's the only sub-module you're talking about here, right?

Comment (steven-sheehy) [reply hedera-mirror-grpc/scripts/deploy.sh]: Probably both. I'd rather not have to write migration to convert it back once it's merged to master.

Comment (apeksharma) [reply hedera-mirror-grpc/scripts/deploy.sh]: Agree with you two. Wanted to make below suggestion earlier, but was okay with anything if this was blocking preview-net deployment.

The separation line between preview-net and prod can be just these two:
1) different instances (obvious)
2) different release version :  To not mix release tars and jars.
Anything more than that, while okay to have, doesn't seem necessary. And it'll be more work undoing them when merging feature branch back to master.
Not forking services (by using different name), unless different release process for hcs is required than our current prod services, will keep things and releases simple.
Would have been great to keep the artifacts name same too by using version=`0.1.0-hcs-rc1` as differentiator (https://gitversion.readthedocs.io/en/latest/git-branching-strategies/gitflow-examples/), but that ship sailed long ago so it's okay.
Basically, as little diverging from norm as possible.

Comment (Nana-EC) [reply hedera-mirror-grpc/scripts/deploy.sh]: Agreed with Appy, to be able to catch migration and deployment issues naming should not reflect their environments. Otherwise you run the risk of hitting environment respective issues.
